qa/tasks/cephfs/filesystem.py

```diff
@@ -640,8 +640,11 @@ def set_down(self, down=True):
     def set_joinable(self, joinable=True):
         self.set_var("joinable", joinable)
 
-    def set_max_mds(self, max_mds):
-        self.set_var("max_mds", "%d" % max_mds)
+    def set_max_mds(self, max_mds, confirm=False):
+        if not confirm:
```

Comment:
reverse if, always prefer avoid using `not X` if an `else:` is present.

Reply:
Also, this change may break numerous tests. I wonder if we should, for our testing, make `confirm=True` the default.

Reply:
No, it doesn't seem to break any existing tests, as `self.fs.set_max_mds(<num>)` is still supported.

Reply:
Added both the changes.